test/binders/propertylinks/GetMyOrganisationPropertyLinksParametersSpec.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package binders.propertylinks
+
+import basespecs.BaseUnitSpec
+import binders._
+import cats.data.Validated.Valid
+
+class GetMyOrganisationPropertyLinksParametersSpec extends BaseUnitSpec {
+
+  import GetMyOrganisationPropertyLinksParameters._
+
+  "validating GetMyOrganisationPropertyLinksParameters" should {
+    "come out VALID" when {
+      "no parameters are provided because all parameters are optional" in {
```

Comment:
are all parameters really optional :O

Reply:
indeed, surprised me but it is what it is, this is how i found it